### ISSUE TITLE
Ordem dos conjuntos de Dados

### ISSUE THREAD

**Issue by silviana-faria** [dados-mg/remuneracao#19]

**Comment by silviana-faria** [comment]
Sugiro que a ordem das planilhas seja em ordem decrescente (mais recente para o mais antigo). Essa regra deverá ser aplicada em todos os conjuntos de dados que a DTA disponibilizar

#### Exemplos:
* [São Paulo](http://dados.prefeitura.sp.gov.br/pt_PT/dataset/remuneracao-servidores-prefeitura-de-sao-paulo)

* [Alagoas](http://dados.al.gov.br/dataset/lei-de-diretrizes-orcamentarias-ldo)

**Comment by fjuniorr** [comment]
Concordo e acho que podemos aplicar isso inclusive na ordenação dos recursos no `datapackage.json`. Fica inclusive mais fácil explicar o comportamento de carga se as duas ordens forem iguais[^1]. 

Para alterar a ordem de recursos já existentes o endpoint correto da API do CKAN parece ser [package_resource_reorder](https://docs.ckan.org/en/2.8/api/index.html#ckan.logic.action.update.package_resource_reorder). Esse [script](https://gist.github.com/nigelbabu/6759804) é antigo mas pode ser outra alternativa.

Não encontrei nada sobre criar em uma ordem pré-determinada.
